ServerCore/Areas/Identity/UserAuthorizationPolicy/AuthorizationHelper.cs

```diff
@@ -75,6 +75,29 @@ private EventRole GetEventRoleFromRoute()
             return EventRole.play;
         }
 
+        public async Task IsPuzzleAdminCheck(AuthorizationHandlerContext authContext, IAuthorizationRequirement requirement)
+        {
+            EventRole role = GetEventRoleFromRoute();
+            if (role != EventRole.admin)
+            {
+                return;
+            }
+
+            PuzzleUser puzzleUser = await PuzzleUser.GetPuzzleUserForCurrentUser(dbContext, authContext.User, userManager);
+            Puzzle puzzle = await GetPuzzleFromRoute();
+            Event thisEvent = await GetEventFromRoute();
```

Comment:
I am aware that this is seemingly unused but it's also unused in the IsPuzzleAuthorCheck method that I copied it from